ksqldb-common/src/main/java/io/confluent/ksql/schema/ksql/SystemColumns.java

```diff
@@ -15,39 +15,49 @@
 
 package io.confluent.ksql.schema.ksql;
 
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
 import io.confluent.ksql.name.ColumnName;
 import io.confluent.ksql.schema.ksql.types.SqlType;
 import io.confluent.ksql.schema.ksql.types.SqlTypes;
+import io.confluent.ksql.util.KsqlException;
+import java.util.HashSet;
+import java.util.Map;
 import java.util.Set;
 
 public final class SystemColumns {
 
   public static final ColumnName ROWKEY_NAME = ColumnName.of("ROWKEY");
-
```

Comment:
nit: unnecessary whitespace change (I think it's slightly more readable with the extra line break since ROWKEY and ROWTIME are distinct system columns).